### DISCUSSION TITLE
Is Outfitter Compat Possible?

### DISCUSSION THREAD

**Discussion by Raylo555** [VickyFrenzy/gmod-enhanced-camera-2#30] [Ideas]
Is it possible to add compatibility with Outfitter? 
I know it's a bit of an odd question, and I would outright request it but I don't even know if it's Possible.

**Comment by VickyFrenzy** [comment]
I'm not really sure. 
I don't personally support Outfitter due to potential security issues with it or other sketchy things the addon is doing...
